apps/vmq_server/test/vmq_cluster_SUITE.erl

```diff
@@ -65,59 +68,69 @@ init_per_testcase(convert_new_msgs_to_old_format, Config) ->
 init_per_testcase(Case, Config) ->
     vmq_test_utils:seed_rand(Config),
     Nodes = vmq_cluster_test_utils:pmap(
-              fun({N, P}) ->
-                      Node = vmq_cluster_test_utils:start_node(N, Config, Case),
-                      {ok, _} = rpc:call(Node, vmq_server_cmd, listener_start,
-                                         [P, []]),
-                      %% allow all
-                      ok = rpc:call(Node, vmq_auth, register_hooks, []),
-                      rpc:call(Node, vmq_subscriber_db, flushall, []),
-                      {Node, P}
-              end, [{test1, 18883},
-                    {test2, 18884},
-                    {test3, 18885}]),
+        fun({N, P}) ->
+            Node = vmq_cluster_test_utils:start_node(N, Config, Case),
+            {ok, _} = rpc:call(
+                Node,
+                vmq_server_cmd,
+                listener_start,
+                [P, []]
+            ),
+            %% allow all
+            ok = rpc:call(Node, vmq_auth, register_hooks, []),
+            rpc:call(Node, vmq_subscriber_db, flushall, []),
+            {Node, P}
+        end,
+        [
+            {test1, 18883},
+            {test2, 18884},
+            {test3, 18885}
+        ]
+    ),
     {CoverNodes, _} = lists:unzip(Nodes),
     {ok, _} = ct_cover:add_nodes(CoverNodes),
-    [{nodes, Nodes},{nodenames, [test1, test2, test3]}|Config].
+    [{nodes, Nodes}, {nodenames, [test1, test2, test3]} | Config].
 
 end_per_testcase(convert_new_msgs_to_old_format, Config) ->
     %% no teardown necessary,
     Config;
 end_per_testcase(_, _Config) ->
-    vmq_cluster_test_utils:pmap(fun(Node) -> ct_slave:stop(Node) end,
-                                [test1, test2, test3]),
+    vmq_cluster_test_utils:pmap(
+        fun(Node) -> ct_slave:stop(Node) end,
+        [test1, test2, test3]
+    ),
     ok.
 
 all() ->
     [
-     multiple_connect_test
-    ,multiple_connect_unclean_test
-    ,distributed_subscribe_test
-    ,racing_connect_test
-    ,racing_subscriber_test
-    ,aborted_queue_migration_test
-    ,cluster_leave_test
-    ,cluster_leave_myself_test
-    ,cluster_leave_dead_node_test
-    ,shared_subs_random_policy_test
-    ,shared_subs_random_policy_online_first_test
-    ,shared_subs_random_policy_all_offline_test
-    ,shared_subs_prefer_local_policy_test
-    ,shared_subs_local_only_policy_test
-    ,cross_node_publish_subscribe
-    ,routing_table_survives_node_restart
-    ,convert_new_msgs_to_old_format
+        multiple_connect_test,
+        multiple_connect_unclean_test,
+        distributed_subscribe_test,
+        racing_connect_test,
+        racing_subscriber_test,
+        aborted_queue_migration_test,
+        cluster_leave_test,
+        cluster_leave_myself_test,
+        cluster_leave_dead_node_test,
+        shared_subs_random_policy_test,
+        %% shared_subs_random_policy_online_first_test,
```

Comment:
This test is skipped due to the change in policy. Can we add a comment mentioning the reason for future reference?